src/network/request/campStatementHistoryRequest.ts

```diff
@@ -14,7 +14,7 @@ export default class CampStatementHistoryRequest extends Request {
       body,
       K.Network.Header.Type.Json,
       {},
-      "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiIyIiwianRpIjoiNDdjOGI5ODg3NTU1MzE4OTliY2U1MjYxY2Q1ZmY4MGVhYTM1NzI2MTU5NzE2YjA3ZjRkOTVhN2U2MzlmNjcyMTNkYTg4NDVhN2M4Mjk4MjAiLCJpYXQiOjE2NTAwMDk2MjQuNDIxMjc3LCJuYmYiOjE2NTAwMDk2MjQuNDIxMjgxLCJleHAiOjE2ODE1NDU2MjQuNDEzODQ1LCJzdWIiOiIzNDEiLCJzY29wZXMiOlsiKiJdfQ.TKgnaQnFWxJcv6iTCOU-ohzRcEjsBB3mb1zjxh5cN3lXNOT0dJTDn9VFKAJjGCHAlXMsk3xKhs4QTA2zC5CtEm8Pekxy4AKtw677c2ZMM850gy2wnUtVdKAK-Y3xJWMVPbfZbuWpCCkRDuN24JLq5tlwZDThf-o8SkBuedlbEmtscKS9CWUcf2SSdeZ9lW2EZfQObzys-FbepejXiw9EYLR-djQrvr0A1iXtPKBnvXtv0YEvA9pxCAlViPEHBrIcD0etMN_GJhgaMxPy1r_wv-67-vLb_QMKuqZ0DP5qICJo8YSZdd4y-3gdQ-QbAgCbn5rEWEQrylISuM2R2yI3OV9l7mX8PhfCRJfO67YjcqzufyZ-lAOXGlku7xEOkBaSxocbR0H1M88CZ6XflgQ8Re03V2264gp-Wa9C3vqP4dtyznnW_8zm9prmoxf24AkEV9UrlhQT3W0l08jbexk4r8_mMomnqrabyu0qYPjro-KoNfQQzE8g_ymtFGz_TTsOMT1RvROv2-nDfyOKznHN1rldvhsmQ6IwCwrYGGnw4jGuiQ6iEPZts0BqpmRhlPLTWiSwfexEsxAOsrAGBhtebKwWTRzib-SZnrI7tlIk7u7LE4SCccSqw_6cOJDzuwbYAiWDh_cQ3x54FTmXvcgFtVAq2fwWZeN1DX_Sk6seOQU"
+      "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiIyIiwianRpIjoiODI2M2NmMjBlODVkMjRhY2EwOTk0MzZlMWNjMWNjNmRjMmE3NDE5YzM1OTIwN2E4ODg0YWY5OTNiNjA5NzI5Yjk4NTJhYzUwOTFmN2JlMjciLCJpYXQiOjE2NTA1MjUwMzEuMzA3MzE1LCJuYmYiOjE2NTA1MjUwMzEuMzA3MzE5LCJleHAiOjE2ODIwNjEwMzEuMjk3MjM2LCJzdWIiOiIxMjMwIiwic2NvcGVzIjpbIioiXX0.P83KeK4hvp63lgPMu64gCfMilA8TfkSWWumga7jZP59gXf3xSPi7MR1FR1M6S02oU52ElSdbkDKAHwgJGbE_siOw-VrYk-PZPJWhdUH5hYAswXSxRZ9aYB1dm6NHzDrTuyS33GKTIm1qQKnFwNGomL7BnuoTys2BU49YVZxVusyiSKRF_vXqDBb39HNsCUCGOO-ZFM3I1iYEVF3EXgg5cHyRTAasKMUFe3RcGk5xDnpls18NVazH4IX6tsQeMI-mqDxAv6XqJ3g-egLMb4kRZcr6b7TwtvXSFXPd4YPCoBQJbeBHpKuWn6QP3UlPXCszgLe4qZI00CKAzpK38V6EkvwHuG_Bflfc8aK8LF4v2Al_8gLal3rh92ZkArk8xu7-553s2ZkOloKRicLp7ckssFKzXuYS5P8RLiLKWrt9OkPAEE7w143W8vIzqKaYc-7g-xQPyrVrfvE2P6pAR7wE0SwlYvvP8pSeYsQEUZEKuHHyOrVnD-Cm8qjngUgYGL3cewSvVwfYOcXZFfN8rYECFsam-BAQ6T3p1inlbDTXVVgqnsrQFZ9oJU9R6Qu6k28WIVlA8eDfu_vQmpW-3mn2IMUoNiTvqxcQ3pt7mnMaL9BdrsjB-uTbkeFJwVn77jIi8WXrYLfM36ag9vDsZFpxmvdlORUxSKwfBefN5BRAadw"
```

Comment:
Looks like a JWT token. Do we need this?

Reply:
Removed